tools/src/commands/SetupReactNativeNightly.ts

```diff
@@ -75,6 +77,17 @@ async function main() {
   await updateBareExpoAsync(nightlyVersion);
 }
 
+async function removePostinstallPatchAsync() {
+  const packageJsonPath = path.join(EXPO_DIR, 'package.json');
+  const packageJson = await JsonFile.readAsync(packageJsonPath);
+  packageJson.scripts = {
+    ...((packageJson.scripts as Record<string, string> | undefined) ?? {}),
+    postinstall:
+      'yarn-deduplicate && yarn workspace @expo/cli prepare && node ./tools/bin/expotools.js validate-workspace-dependencies',
+  };
```

Comment:
Why not just this 👇?

```suggestion
  packageJson.scripts.postinstall = 'yarn-deduplicate && yarn workspace @expo/cli prepare && node ./tools/bin/expotools.js validate-workspace-dependencies';
```

Reply:
because of typescript treat `packageJson.scripts` as JSONValue | undefined type, we cannot simply assign that.
will keep it as-is since we will remove this soon when we remove the postinstall patches.